JS/Lesson15/salih/script.js

```diff
@@ -133,14 +174,96 @@ checkIfStringIsUpperCase('write a program that takes in');
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
 
+function chechBMI(height, weight){
+	const BMI = weight / ([height]^2);
+	if (BMI <= 18.5){
+		return "Underweight";
+	} else if (BMI > 18.5 && BMI <24.9){
+		return "Healthy Weight";
+	} else if (BMI > 25 && BMI <29.9) {
+		return "Overweight";
+	} else if (BMI > 30) {
+		return "Above Obesity";
+	} else {
+		return "N/A"
+	};
+}
+console.log('Exercise 8.1:', chechBMI(1.80, 95));
+console.log('Exercise 8.2:', chechBMI(1.80, 82));
+console.log('Exercise 8.3:', chechBMI(1.80, 70));
+console.log("__________________________________________________");
+
 // 9. Write a program that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
 
+function checkAge(age){
+	if (age > 0 && age < 12){
```

Comment:
You need to include 12, 19, 20, 64, and 65 too
```suggestion
	if (age > 0 && age < =12){
```